.github/workflows/ci-tests.yml

```diff
@@ -85,6 +85,8 @@ jobs:
       run: |
         echo "TOXENV=${{ matrix.jobs }}" >> $GITHUB_ENV
     - name: Run the tests
+      env:
+        SQLFLUFF_BENCHMARK_API_KEY: {{ secrets.SQLFLUFF_BENCHMARK_API_KEY }}
```

Comment:
This line is stopping the CI checks. Does it need a `$` before the `{{` as per the syntax here? https://docs.github.com/en/actions/learn-github-actions/contexts#env-context

Reply:
@alanmcruickshank you're correct, I've made the update now

Reply:
Checks look good